openstack/utils/templates/_proxysql.tpl

```diff
@@ -63,6 +63,7 @@
     {{- include "utils.proxysql.volume_mount" . | indent 2 }}
       {{- end }}
     {{- end }}
+  {{- include "utils.trust_bundle.volume_mount" . | indent 2 }}
```

Comment:
> Defaulting to enabled, as the charts need to be adapted on a per service-level anyway, and is more an option to disable it, in case there is a problem discovered after deployment.

With this line here, every chart updating the utils chart will automatically get one trust-manager volume mount in (basically) every pod, while it's not clear that the chart is including the volume definition. I suspect this will lead to problems. Can we include the `utils.trust_bundle.volumes` somehow (with another name) to make it work or do you think every chart will be adapted before upgrading to utils `0.12.0`?

Reply:
My idea was update all the charts before upgrading to `utils>=0.12.0`.
Yes, it will break things, but it would fail quite early in the pipeline when trying to apply the chart, not somewhere down the pipeline.

I think, we should avoid keeping too many variants of how things are done around, we should rather focus our effort on making things more equal more quickly.

But you are right, that might block changes on utils and requires syncing with people on that.
I'll default to disable it, and enable it in the values files with the changes to the charts using it in the values.

Reply:
Updated the commit message accordingly as well.